src/pages/Debug/Report/DebugReportActions.tsx

```diff
@@ -37,10 +37,7 @@ function DebugReportActions({reportID}: DebugReportActionsProps) {
         </PressableWithFeedback>
     );
     return (
-        <ScrollView
-            style={styles.mt5}
-            contentContainerStyle={styles.pb5}
-        >
+        <View style={[styles.mv5, styles.flex1]}>
```

Comment:
NAB: But we have a point on the checklist

>  If a new page is added, I verified it's using the ScrollView component to make it scrollable when more elements are added to the page

Here, we use Flatlist instead of ScrollView. But I think It still make sense

Reply:
@DylanDylann Thanks for pointing this out! I just applied added ScrollView again and disabled scroll on the FlatList and the problem is gone 😄 

Let me know if you like this solution!